test/canary-environment/models/table/table.sql

```diff
@@ -0,0 +1,12 @@
+-- Copyright Materialize, Inc. and contributors. All rights reserved.
+--
+-- Use of this software is governed by the Business Source License
+-- included in the LICENSE file at the root of this repository.
+--
+-- As of the Change Date specified in that file, in accordance with
+-- the Business Source License, use of this software will be governed
+-- by the Apache License, Version 2.0.
+
+{{ config(materialized='table') }}
+
+CREATE TABLE {{ this }} (c int);
```

Comment:
I think it is important to create a materialized view on top of this table , plus an index, just for the sake of putting load on the compute layer as well.